tests/unit/backends/test_cloud_monitoring_mql.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# flake8: noqa
+
+import unittest
+
+from slo_generator.backends.cloud_monitoring_mql import \
+    CloudMonitoringMqlBackend
+
+
+class TestCloudMonitoringMqlBackend(unittest.TestCase):
```

Comment:
I was not sure how and where to put these unit tests in the existing suite. I wrote a dedicated ets file for the new backend.